### DISCUSSION TITLE
Why can't I use the command YOLO?

### DISCUSSION THREAD

**Discussion by KaedeharaT** [ultralytics/ultralytics#3406] [Q&A]
Hello, I'm new to GitHub and programming. I've recently been using YOLOv8 to work on my research project. The command "yolo predict model=yolov8n.pt source='https://ultralytics.com/images/bus.jpg'" runs smoothly and produces the desired output on my desktop. However, when I attempt to run the same command on my laptop, I receive the error message: "yolo: The term 'yolo' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again." I suspect that this issue may be a result of incorrect environment setup. Could you please assist me with resolving this problem? Thank you very much.
![111](https://github.com/ultralytics/ultralytics/assets/110965108/c1fb17f7-983b-40b8-a4b8-8e7a032519cf)

**Comment by kholidiyah** [comment]
I can solve by updating ultralytics

pip install --upgrade ultralytics

**Reply by pderrenger** [reply]
@kholidiyah it seems like you're encountering an issue with the command-line interface not recognizing the `yolo` command. This could be due to the `ultralytics` package not being installed or not properly set up in your laptop's environment. To resolve this, you can try the following steps:

1. Ensure that Python and pip are correctly installed on your laptop.
2. Install the `ultralytics` package by running the command:
   ```python
   pip install --upgrade ultralytics
   ```
3. After installation, try running the `yolo` command again.

If you continue to face issues, please refer to the [Ultralytics Docs](https://docs.ultralytics.com) for more detailed instructions on installation and usage. 😊👍
